embrace-android-api/src/main/kotlin/io/embrace/android/embracesdk/spans/TracingApi.kt

```diff
@@ -264,4 +264,6 @@ public interface TracingApi {
      * if was completed in a prior session.
      */
     public fun getSpan(spanId: String): EmbraceSpan?
+
+    public fun createOperation(name: String): Workflow
```

Comment:
I'd suggest not putting this on the main tracing API as it's effectively syntactic sugar on top of that